internal/xsql/conn.go

```diff
@@ -362,3 +366,354 @@ func (c *conn) BeginTx(ctx context.Context, txOptions driver.TxOptions) (_ drive
 	}
 	return c.currentTx, nil
 }
+
+func (c *conn) Version(ctx context.Context) (_ string, err error) {
+	const version string = "default"
+	return version, nil
+}
+
+func (c *conn) IsTableExists(ctx context.Context, tableName string) (tableExists bool, err error) {
+	cn := c.connector.Connection()
```

Comment:
Please replace all `cn` vars to `cc`. For single style of code

Reply:
@asmyasnikov I fixed, please check again